store/src/commonMain/kotlin/org/mobilenativefoundation/store/store5/Converter.kt

```diff
@@ -1,47 +1,45 @@
 package org.mobilenativefoundation.store.store5
 
-interface Converter<Network : Any, Output : Any, Local : Any> {
-    fun fromNetworkToOutput(network: Network): Output?
-    fun fromOutputToLocal(output: Output): Local?
-    fun fromLocalToOutput(local: Local): Output?
-
-    class Builder<Network : Any, Output : Any, Local : Any> {
-
-        private var fromOutputToLocal: ((output: Output) -> Local)? = null
-        private var fromNetworkToOutput: ((network: Network) -> Output)? = null
-        private var fromLocalToOutput: ((local: Local) -> Output)? = null
-
-        fun build(): Converter<Network, Output, Local> =
-            RealConverter(fromOutputToLocal, fromNetworkToOutput, fromLocalToOutput)
-
-        fun fromOutputToLocal(converter: (output: Output) -> Local): Builder<Network, Output, Local> {
+/**
+ * Converter is a utility interface that can be used to convert a network or output model to a local model.
+ * Network to Model conversion is needed when the network model is different what you are saving in
+ * your Source of Truth.
+ * Output to Local conversion is needed when you are doing local writes in a MutableStore
+ * @param Network the network model type
+ * @param Output the output model type
+ * @param Local the local model type
+ */
+interface Converter<Network : Any, Local : Any, Output : Any> {
+    fun fromNetworkToLocal(network: Network): Local
+    fun fromOutputToLocal(output: Output): Local
```

Comment:
My understanding is `OutputToLocal` is only used with `MutableStore`. Should this interface be separated?